docs/components/time_pickers.md

```diff
@@ -0,0 +1,329 @@
+**Shape** | `dayPeriodShape` property in `TimePickerThemeData`
+**Color** | `dayPeriodColor` property in `TimePickerThemeData`
+**Text Color** | `dayPeriodTextColor` property in `TimePickerThemeData`
+**Text Style** | `dayPeriodTextStyle` property in `TimePickerThemeData`
+
+
+### Mobile time input pickers
+
+Mobile time input pickers allow users to specify time using the numbers on a keyboard. The mobile time input picker can be accessed from any other mobile time picker interface by tapping the keyboard icon
+
+#### Mobile time input picker example
+
+`showTimePicker`
+- [Class definition](https://api.flutter.dev/flutter/material/showTimePicker.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/time_picker.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/time_pickers/input&gh_ref=develop)
+
+The following example shows a time input picker.
+
+![Time input picker](assets/time_pickers/time_input_picker_example.png)
+
+```dart
+final TimeOfDay newTime = await showTimePicker(
+  context: context,
+  initialTime: TimeOfDay(hour: 7, minute: 15),
+  initialEntryMode: TimePickerEntryMode.input,
+);
+```
+
+#### Anatomy and key properties
+
+![Mobile time input picker anatomy diagram](assets/time_pickers/input_anatomy.png)
+
+1. Title
+1. Hour and minute input field
+1. Icon button to switch to dial entry
+1. AM/PM selector
+
+##### Title
+
+&nbsp;         | Property
+-------------- | ------------------------ 
+**Text** | `helpText` property in `showTimePicker`
+**Text Style** | `helpTextStyle` property in `TimePickerThemeData`
+
+##### Hour and minute input
+
+&nbsp;         | Property
+-------------- | ------------------------ 
+**Style** | `inputDecorationTheme` property in `TimePickerThemeData`
+
+##### Input type toggle
+
+&nbsp;         | Property
+-------------- | ------------------------ 
+**Color** | `entryModeIconColor` property in `TimePickerThemeData`
+
+##### AM/PM selector
+
+&nbsp;         | Property
+-------------- | ------------------------ 
+**Border** | `dayPeriodBorderSide` property in `TimePickerThemeData`
+**Shape** | `dayPeriodShape` property in `TimePickerThemeData`
+**Color** | `dayPeriodColor` property in `TimePickerThemeData`
+**Text Color** | `dayPeriodTextColor` property in `TimePickerThemeData`
+**Text Style** | `dayPeriodTextStyle` property in `TimePickerThemeData`
+
+
+## Theming time pickers
+
+The following example shows a mobile time picker that uses the [Shrine theme](https://material.io/design/material-studies/shrine.html):
+
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/time_pickers/theme&gh_ref=develop)
+
+![Time Picker with Shrine theme](assets/time_pickers/time_picker_theme_example.png)
```

Comment:
Update alt-text to:

"Time picker set to 7:15AM, selected hour in pink and other text in brown."